sql/hive-thriftserver/pom.xml

```diff
@@ -41,11 +41,8 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.apache.spark</groupId>
-      <artifactId>spark-core_${scala.binary.version}</artifactId>
-      <version>${project.version}</version>
-      <type>test-jar</type>
-      <scope>test</scope>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-servlet</artifactId>
```

Comment:
Thanks for clarifying @srowen , will change from provided to compile scope.
Since build fails with '-Dmaven.test.skip=true' without the dependency, I will add it as part of this PR itself.

Assuming no github issues, do let me know of any other changes/comments, thanks !